tencentcloud/resource_tc_instance.go

```diff
@@ -303,11 +303,11 @@ func resourceTencentCloudInstanceCreate(d *schema.ResourceData, m interface{}) e
 		maxBandwidthOut := internetMaxBandwidthOut.(int)
 		params["InternetAccessible.InternetMaxBandwidthOut"] = fmt.Sprintf("%v", maxBandwidthOut)
 	}
+	// set public ip assignment default to false, assign public IP when `allocate_public_ip` explicitly set to true.
+	params["InternetAccessible.PublicIpAssigned"] = "FALSE"
```

Comment:
This will change default behavior, if user don't specify this argument, it should not pass FALSE to API, and its default value is "TRUE" in API side. Please put this line inside below **if** block

Reply:
```golang
...
"allocate_public_ip": &schema.Schema{
    Type:     schema.TypeBool,
    Optional: true,
    Default:  false,
},
...
```
This is based on the schema definition which set the `allocate_public_ip` default to false.

Reply:
So you mean I should change the default value of `allocate_public_ip` to true which conforms the API default value.

```go
"allocate_public_ip": &schema.Schema{
    Type:     schema.TypeBool,
    Optional: true,
    Default:  true,
},
```